src/test/java/org/folio/qm/controller/RecordsEditorRecordsApiTest.java

```diff
@@ -284,13 +296,68 @@ void testReturn400IfQmRecordIdIsEmpty() {
   }
 
   @Test
-  void testPostQuickMarkValidRecordAccepted() {
+  void testPostQuickMarkValidRecordCreated() throws IOException {
```

Comment:
```suggestion
  @ClearTable(RECORD_CREATION_STATUS_TABLE_NAME)
  void testPostQuickMarkValidRecordCreated() throws IOException {
```

Reply:
Will clear table after the test execution

Reply:
added